project_finder_web_app/models.py

```diff
@@ -1,41 +1,73 @@
 from django.db import models
 from django.utils import timezone
+from django.contrib.auth.models import AbstractUser
 # Create your models here.
 
 
+class Skill(models.Model):
+    skill = models.CharField(max_length=50, unique=True)
+
+    def __str__(self):
+        return self.skill
+
+
+class User(AbstractUser):
+    email = models.EmailField(unique=True)
+    sap_id = models.CharField(max_length=15, unique=True)
+    mobile = models.CharField(max_length=10, unique=True)
+    photo = models.FileField(upload_to='certain_location', blank=True, null=True)
+    is_mentor = models.BooleanField(default=False)
+    is_teacher = models.BooleanField(default=False)
+    bio = models.TextField(max_length=500, blank=True)
+    Github = models.URLField(null=True, blank=True)
+    LinkedIN = models.URLField(null=True, blank=True)
+    years = (
+        ('FE', 'First Year'),
+        ('SE', 'Second Year'),
+        ('TE', 'Third Year'),
+        ('BE', 'Fourth Year'),
+    )
```

Comment:
Add a choice for Alumnus/Alumni. BEs jayenge abbhi, aisa mat karo unlogo ke saath.